authorization_services/topics/policy-evaluation-api.adoc

```diff
@@ -5,8 +5,8 @@ When writing rule-based policies using JavaScript or JBoss Drools, {project_name
 
 This API consists of a few interfaces that provides you access to information such as:
 
-* The permission being requested
-* The identity that is requesting the permission, from which you can obtain claims/attributes
+* The permission being evaluated, representing both the resource and scopes being requested.
+* Obtain attributes associated with the resource being requested
```

Comment:
The attributes associated with...

(Because the introduction to the list already says that the API provides access to information, such as...)